addons/account/models/account_move_send.py

```diff
@@ -381,7 +381,7 @@ def _link_invoice_documents(self, invoices_data):
         if not attachment_to_create:
             return
 
-        attachments = self.env['ir.attachment'].create(attachment_to_create)
+        attachments = self.sudo().env['ir.attachment'].create(attachment_to_create)
```

Comment:
We probably should avoid doing this on data we don't control here. Like do it in the parent (of parent I think) where we manipulate the move to create the invoices_data.
This way, we won't surprise someone with this.

I don't think it changes anything big...